### PR TITLE
Add flag for DevMode to Docmosis

### PR DESCRIPTION
**Pull request by tchow8**
# Description

[DIV-5257](https://tools.hmcts.net/jira/browse/DIV-5257)

This will prevent failing Docmosis templates from breaking the build on AAT.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: infrastructure/aat.tfvars [hunk 1]
```diff
@@ -3,3 +3,5 @@ capacity = "2"
 instance_size = "I3"
 
 health_check_ttl = 30000
+
+docmosis_dev_mode_flag = true
```
Comment (qzhou-hmcts) [inline infrastructure/aat.tfvars]: string or boolean?

File: infrastructure/demo.tfvars [hunk 2]
```diff
@@ -1,2 +1,4 @@
 vault_env = "preprod"
 capacity = "2"
+
+docmosis_dev_mode_flag = true
```
Comment (qzhou-hmcts) [inline infrastructure/demo.tfvars]: string or boolean?

File: infrastructure/preview.tfvars [hunk 3]
```diff
@@ -2,3 +2,5 @@ vault_env = "preprod"
 
 health_check_ttl = 30000
 feature_resp_solicitor_details = "true"
+
+docmosis_dev_mode_flag = true
```
Comment (qzhou-hmcts) [inline infrastructure/preview.tfvars]: This is a string or a boolean?

Comment (tchow8) [reply infrastructure/preview.tfvars]: Nice catch, chaning to string

**Comment by nsingamsetty** [comment]
Merging this for failing Docmosis Templates

**Pull request merged**